sectioning.asciidoc

```diff
@@ -16,4 +22,24 @@ Chapters should have two equals signs.  This turns into an +<h2>+ tag in html
 If the book is broken up into smaller subsections than chapters, you can use more equal signs:
 
   == Chapter 9
-  === Section A
\ No newline at end of file
+  === Section A
+
+If a chapter or section heading has a title as well as a number, put them all on the same line and separated by a colon:
+
+  == Chapter 5: Advice from a Caterpillar
+
+== Pages
+Where a Gitenberg book has scans of the original available,
+you can choose to include page-markers (http://asciidoc.org/userguide.html#X30[anchors] in Asciidoc)
+to make it easier to relate the text to the original images.
+These are invisible in the formatted output.
+
+  First came ten soldiers carrying clubs; these were all shaped like the three
+  gardeners, oblong and flat, with their hands and feet at the corners: next the
+  ten courtiers; these were ornamented all over with diamonds, and walked two
+  and [[page-115]] two, as the soldiers did. After these came the royal children;
+  there were ten of them, and the little dears came jumping merrily along hand in
+  hand, in couples: they were all ornamented with hearts. 
+
+The anchor should be of the form `[[page-x]]` where `x` is the number of the page.
```

Comment:
Yes I see what you mean, although I can't think of any actual collisions. I'll change it to `[[source-page-x]]` because it's conceivable that the text has never actually been OCR'd.